Assets/_Dev/Scripts/Scene/Scene_Title.cs

```diff
@@ -4,8 +4,44 @@
 
 public class Scene_Title : MonoBehaviour
```

Comment:
class 이름에는 _를 쓰면 안됩니다.
```suggestion
public class SceneTitle : MonoBehaviour
```

Reply:
넵, 예전에 작성했던 클래스명인데 앞으로 코드작업 할 때 해당부분들이 있으면 같이 수정토록 하겠습니다!